src/generators/crud/default/views/index.php

```diff
@@ -134,7 +134,15 @@
             $model = new $generator->modelClass();
             ?>
             <?php foreach ($generator->getModelRelations($model) as $relation): ?>
+
                 <?php
+                $isVisibleCondition = true;
+                $relationName = StringHelper::basename($relation->modelClass);
+                if ($generator->accessFilter) {
```

Comment:
Could that login be moved to the generator instead of the view?

Reply:
This needs to be done for every relation. The relationships come from a [separate trait](https://github.com/schmunk42/yii2-giiant/blob/master/src/generators/crud/ModelTrait.php#L71) I see what you mean, but at this point I think it's justifiable to do this check in the view as we only need it at this one place.